### ISSUE TITLE
Create a Confirm Order button.

### ISSUE THREAD

**Issue by yigiaom** [cse442-at-ub/project_s23-one-belt-one-road#27]
**Task tests**

*Test 1*
1. Open the e-commerce website, click on "Log in" button (username: abcd, password: 12345678).
2. Click on the image of hat on the landing page.
3. Click on "Add to Cart" button on the item page.
4. Click the button in the shape of a shopping cart in the upper right corner.
5. On the shopping cart page, click on the "Check Out" button and fill in your payment information.
6. On payment page, verify if there is a "Confirm Order" button.

*Test 2*
1. Go to https://www.figma.com/file/qXW0S7J0YT48satDEr5l5B/Pages?node-id=0%3A1&t=pFo3MZFdawZrKbwd-0
2. On the Payment Page, verify if there is a "Confirm Order" button.
3. In the HTML code, make certain the button has an id of "btnConfirmOrder".

**Comment by anht-11** [comment]
"If a task is added to the Planned pipeline in one sprint, but does not get developed in that time, DO remove the original sprint label and replace it with the label from the new sprint." - Style Guide

A task should block a user story only, not a task, so you might consider removing the task being blocked by this one.

Also, I suggest we group all tasks related to the confirmation page to one task only, so it would be easy to run multiple task tests on one issue, instead of jumping around tasks.

**Comment by yigiaom** [comment]
This task has to be closed because we integrate all tasks related to the payment page to one task.
